### ISSUE TITLE
Could not parse version: line is '// TypeScript Version: 2.4'

### ISSUE THREAD

**Issue by BjornMelgaard** [Microsoft/dtslint#78]
Hi, I'm trying to move typings from `@types/ramda` to `ramda`
I've done everything like in readme, but getting this

```
 ~/projects/ramda   feature/integrate_ts_typings ●  yr dtslint
yarn run v1.1.0
$ dtslint types
Could not parse version: line is '// TypeScript Version: 2.4'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

[corresponding commit](https://github.com/BjornMelgaard/ramda/tree/52b5a9a50466c4baab08b102f106f37847194abf)

**Comment by cameron-martin** [comment]
I'm getting this error for anything over TypeScript 2.2

**Comment by BjornMelgaard** [comment]
@cameron-martin seems like dtslint is working only in DefenitelyTyped, one possible workaround is to use dts-jest in non-DefenitelyTyped projects like it made in https://github.com/types/npm-ramda

**Comment by corydeppen** [comment]
This is a blocker for anyone trying to use current React definitions that make use of default type arguments that have been supported since TS 2.3. Considering TS 2,5 was released nearly two months ago, I'm confused why it's not possible to compile using at least TS 2,3.

**Comment by andy-ms** [comment]
Should be fixed with the 0.2 release.
